packages/tracing/src/browser/browsertracing.ts

```diff
@@ -95,6 +103,7 @@ export class BrowserTracing implements Integration {
 
   /** Browser Tracing integration options */
   public options: BrowserTracingOptions = {
+    enableMeasurements: false,
```

Comment:
We should probably make this default `true`, also fix here then https://github.com/getsentry/sentry-javascript/pull/2909/files#diff-a36c2a8cc4a07fe73d56842b30b4a1dfR26

Reply:
@HazAT Measurements will be exposed to internal and a limited set of customers. Would that be fine to enable by default?

Reply:
Yeah I guess it's fine, it doesn't increase the payload significantly